tests/tap.sh

```diff
@@ -36,35 +36,27 @@ for ((i=0;i<${#backends[@]}; ++i)); do
     # grep to skip test if backend cannot handle resource
     if grep -F -q -e 'backend cannot use resource' \
             -e 'OCCA backend failed' ${output}.err; then
-        printf "ok $i0 # SKIP $1 $backend\n"
-        printf "ok $i1 # SKIP $1 $backend stdout\n"
-        printf "ok $i2 # SKIP $1 $backend stderr\n"
+        printf "ok $i0 # OCCA SKIP $1 $backend\n"
```

Comment:
I believe [TAP](http://testanything.org/tap-specification.html#skipping-tests) says the `SKIP` must come first.

Reply:
Correct, fixed